lib/Extension/Bundle/Traits/PackageVersionTrait.php

```diff
@@ -27,12 +27,25 @@
  */
 trait PackageVersionTrait
 {
+
     /**
      * Returns the composer package name used to resolve the version
      *
      * @return string
      */
-    abstract protected function getComposerPackageName(): string;
+    public function getComposerPackageName(): string {
+        foreach([__DIR__ . '/../composer.json', __DIR__ . '/composer.json'] as $composerFile) {
+            if (file_exists($composerFile)) {
```

Comment:
Maybe use `is_file` instead of `file_exists` (which also returns true for directories)?

```suggestion
            if (is_file($composerFile)) {
```